SushiBeltTrackerItemRect.swift

```diff
@@ -0,0 +1,32 @@
+//
+//  SushiBeltTrackerItemRect.swift
+//  SushiBelt
+//
+//  Created by Jaxtyn on 2022/08/16.
+//
+
+import Foundation
+
+#if !os(macOS)
+
+import UIKit
+
+typealias PlatformView = UIView
+
+#else
+
+import AppKit
+
+typealias PlatformView = NSView
+
+#endif
+
+public struct SushiBeltTrackerItemRect {
+  let frameInWindow: CGRect
+}
```

Comment:
How about make a public initialization such as
```swift
init(x: Float, y: Float, width: Float, height: Float) { ... }
init(x: Int, y: Int, width: Int, height: Int) { ... }
```
is it too much?

Reply:
I also think it's good to have a `public init`

Reply:
good opinion applied it
thx :)

https://github.com/daangn/SushiBelt/pull/9/commits/f6e8e141746110ed107a637f1684a27e817e11e8